src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: 200px;
+  height: 410px;
```

Comment:
you shouldn't set height for card, elements inside should set it
If you want to have height 410px so make the content to have this height
Like in  Math, sum of height of elements inside should give you 410px